base/scene.py

```diff
@@ -22,5 +23,5 @@ def render(self):
     def handle_input(self):
         pass
 
-    def transition_to(self, scene_name):
-        self.scene_manager.transition_to(scene_name)
+    def transition_to(self, scene_name, **kwargs):
```

Comment:
I am not a fan of passing `**kwargs` around if not needed.  

Taking a step back it looks like you are trying to [pull the player out of the `kwargs` in the InventoryScene](https://github.com/jpalladino84/Python-Roguelike-Framework/pull/29/files#diff-6bb8636eee43c77c769e3c98558472edR31) but the [player object should be in the game_context already](https://github.com/jpalladino84/Python-Roguelike-Framework/blob/master/managers/game_context.py#L8).